### PR TITLE
Execution model grammar fixes

### PR DESCRIPTION
**Pull request by jdahm**
Minor grammar fixes found when reading through.

### CODE REVIEW

File: docs_src/manuals/user_manual/execution_model.hrst [hunk 1]
```diff
@@ -3,35 +3,31 @@
...
 - `forward`: The computation at index `k` in the vertical dimension is executed after index `k - 1` for all points in
   the horizontal plane;
 - `backward`: The computation at index `k` in the vertical dimension is executed after index `k + 1` for all points in
   the horizontal plane;
 - `parallel`: No order is specified and execution can happen concurrently.
 
-More concretely, a multistage is a list of stages (implemented with stencil operators) to be executed with a certain
-execution policy.  A computation combines several multistages, and will execute one multistage after the other.
+More concretely, a multistage is a list of stages, implemented with stencil operators, to be executed with a certain
+execution policy. A computation combines several multistages and is executed one multistage after the other.
```
Comment (havogt) [inline docs_src/manuals/user_manual/execution_model.hrst:19]:
```suggestion
execution policy. A computation combines several multistages and executes one multistage after the other.
```
right?

Comment (jdahm) [reply docs_src/manuals/user_manual/execution_model.hrst:19]: Either way sounds good, but I prefer your way. I'll accept the fix!

File: docs_src/manuals/user_manual/execution_model.hrst [hunk 2]
```diff
@@ -3,35 +3,31 @@
...
+  stage can assume that the previous stage has been applied to the the whole `IJ` plane before its execution. The user
+  can explicitly create independent stages that do not require this restriction.
 - If the execution policy is `parallel`, a stage cannot impose any assumptions on which stages are applied before in
   another `IJ` plane.
-- If the execution policy is `forward`, it is guaranteed, that if a stage is executed at index ``k``,
-  then all previous stages of
-  the multistage were already applied to the same column with smaller ``k``. There is no guarantee that previous stages
-  of the multistage are
+- If the execution policy is `forward`, it is guaranteed that if a stage is executed at index ``k`` then all stages of
+  the multistage were already applied to the same column with smaller ``k``, and that no stages of the multistage are
```
Comment (fthaler) [inline docs_src/manuals/user_manual/execution_model.hrst:27]: This is wrong, depending on the backend (simple example: the naive one), previous stages of the multistage _are_ already applied on the same column with larger k.

File: docs_src/manuals/user_manual/execution_model.hrst [hunk 3]
```diff
@@ -3,35 +3,31 @@
...
   already applied to the indices in the same column with larger ``k``.
-- If the execution policy is `backward`, it is guaranteed, that if a stage is executed at index ``k``,
-  then all previous stages of
-  the multistage were already applied to the same column with larger ``k``. There is no guarantee that previous stages
-  of the multistage are
+- If the execution policy is `backward`, it is guaranteed that if a stage is executed at index ``k`` then all stages of
+  the multistage were already applied to the same column with larger ``k``, and that no stages of the multistage are
```
Comment (fthaler) [inline docs_src/manuals/user_manual/execution_model.hrst:30]: Same here…

File: docs_src/manuals/user_manual/execution_model.hrst (hunk 2 above)
Comment (jdahm) [reply docs_src/manuals/user_manual/execution_model.hrst:27]: Good catch. Will fix now.

**Comment by jdahm** [comment]
@fthaler Thanks for taking a look, and sorry for the delay. I need to re-adjust GitHub notifications... This should be ready for a final review then.

**Comment by anstaf** [comment]
launch jenkins

**Pull request merged**